.github/ISSUE_TEMPLATE/blocking_bug_report.yml

```diff
@@ -49,14 +49,10 @@ body:
           required: false
         - label: Windows amd64
           required: false
-        - label: Windows arm64
```

Comment:
The amd64 pyOCD installer should work on arm64 thru the compat layer in Windows.

Reply:
Let's remove for the moment until we had the opportunity to test this.